pulumi/src/cluster/docker.ts

```diff
@@ -0,0 +1,64 @@
+import { hashElement } from "folder-hash"
+import { Dockerhub } from ".."
+import { hasTag, buildAndPushImage } from "../docker"
+import { getBaseHash, getVolumeReaperHJash as getVolumeReaperHash } from "../hasher"
```

Comment:
don't forget this too
```suggestion
import { getBaseHash, getVolumeReaperHash } from "../hasher"
```